### PR TITLE
Asset holding

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Proposed Changes

+ adding new function to return asset holding and use it in `balanceOf`

### CODE REVIEW

File: packages/web/src/lib/status.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import type { Algodv2, modelsv2 } from "algosdk";
+
+/**
+ * Returns `account` balance of `assetID`. Returns 0 if the account has not
+ * opt-in to the given asset id.
+ * @param deployer algob deployer
```
Comment (ratik21) [inline packages/web/src/lib/status.ts:6]: this param should be `algodClient`

File: packages/web/src/lib/status.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import type { Algodv2, modelsv2 } from "algosdk";
+
+/**
+ * Returns `account` balance of `assetID`. Returns 0 if the account has not
+ * opt-in to the given asset id.
```
Comment (ratik21) [inline packages/web/src/lib/status.ts:5]:
```suggestion
 * opted-in to the given asset id.
```

**Review by ratik21 — APPROVED** [review]

File: packages/web/src/lib/status.ts (hunk 1 above)
Comment (robert-zaremba) [reply packages/web/src/lib/status.ts:6]: updated

File: packages/web/src/lib/status.ts (hunk 2 above)
Comment (robert-zaremba) [reply packages/web/src/lib/status.ts:5]: I've updated to "did not opt-in"

**Pull request merged**